Dockerfile

```diff
@@ -498,11 +506,16 @@ RUN \
 # https://gitlab.alpinelinux.org/alpine/aports/-/issues/11806
 RUN sed -i 's/-lgcc_s//' /usr/local/lib/pkgconfig/rav1e.pc
 
+# sed to fix symbol name conflict with libssh (md5_init, md5_append, md5_finish)
```

Comment:
A bit annoying but seems like they are aware of it was least https://github.com/Haivision/srt/issues/443 https://github.com/Haivision/srt/issues/1924 maybe add as reference?

Reply:
didn't know, i didn't look... simplest for them would be to just use a `namespace libsrt {` in the code..